packages/server/src/controllers/UsersController.js

```diff
@@ -4,7 +4,135 @@ import UsersValidator from '../validators/UsersValidator'
 
 const { User, UserAddress } = Models
 
+// temp dummy models
+let Users = { }
+let UserEmails = { }
+let UserSessions = { }
+
+
 export default {
+  async details(req, res) {
+    let body = {
```

Comment:
Generally prefer `const` over `let`, unless you know the variable is going to change :)